build_images.sh

```diff
@@ -1,31 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
This file should also not change. Please make it consistent with master branch in this github repo